test/tests/time_steppers/iteration_adaptive/tests

```diff
@@ -93,4 +93,12 @@
     requirement = "MOOSE shall support the ability to reject a time step based
     on a threshold value for the ratio of the ideal step size to the limit."
   [../]
+  [./test_piecewise_linear]
+    type = 'Exodiff'
+    input = 'piecewise_linear.i'
+    exodiff = 'piecewise_linear_out.e'
```

Comment:
It would probably be better to just do a csvdiff for this one since the only thing you really want to compare is the value of dt.

Reply:
Done.